src/DDoSX/AclGeoIpClient.php

```diff
@@ -0,0 +1,48 @@
+<?php
+namespace UKFast\SDK\DDoSX;
+
+use UKFast\SDK\Client as BaseClient;
+use UKFast\SDK\DDoSX\Entities\AclGeoIpRule;
+use UKFast\SDK\SelfResponse;
+
+class AclGeoIpClient extends BaseClient
+{
+    /**
+     * @inheritDoc
+     */
+    protected $basePath = 'ddosx/';
+
+    const RULE_MAP = [];
+
+    /**
+     * Update a ACL GeoIp Rule for a domain
+     * @param $domainName
+     * @param $geoIpRuleId
+     * @param $data
+     * @return SelfResponse
+     * @throws \GuzzleHttp\Exception\GuzzleException
+     */
+    public function updateRule($domainName, $geoIpRuleId, $data)
```

Comment:
sanity check: this includes `rule` because the acl geoip endpoint also deals with mode

Reply:
I'd rename `$data` to `$geoIpRule` and include the object for easier typehinting